client/src/locales/en-US.json

```diff
@@ -662,13 +662,14 @@
             "moving": "Moving..."
         },
         "fileContextMenu": {
-            "titleManage": "Manage file",
+            "titleManageFile": "Manage file",
+            "titleManageFolder": "Manage folder",
```

Comment:
Doesn't match the french version. Choose either with a verb `Manage file|folder` / `Gérer un fichier|dossier` or without (my personal pick) `File|Folder management` / `Gestion du fichier|dossier`